<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1901</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1901</width>
     <height>991</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget
{  
background-color: rgb(69, 191, 81);
 }
</string>
   </property>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;

font: 25pt &quot;Showcard Gothic&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>1441</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 108));
border-style: solid;
border-width: 5px;
border-color: rgb(255, 255, 255);
border-radius: 15px;

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25itms">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>351</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_50itms">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>160</y>
      <width>351</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="download25itms">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>780</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download50itms">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>780</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>-20</y>
      <width>1131</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(85, 85, 255);
font: 75 50pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Get Your Template In Seconds</string>
    </property>
   </widget>
   <zorder>graphicsView_3</zorder>
   <zorder>label_50itms</zorder>
   <zorder>label_25itms</zorder>
   <zorder>back_button</zorder>
   <zorder>download25itms</zorder>
   <zorder>download50itms</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
